### PR TITLE
Write to doors

### PR DESCRIPTION
**Pull request by asavu91**
- implemented a class WriteInfo that inherits from the Action class and provides functionality to write data
- created a JavaScript function writeInfoDoors which prompts the user to insert door parameter.
- created the POST endpoint in flask.

### CODE REVIEW

**Review by CatalinGP — CHANGES_REQUESTED** [review]
review my code comments.

File: src/rest_api/actions/write_info_action.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        id = self.my_id * 0x100 + id_mcu
+
+        try:
+            log_info_message(logger, "Changing session to default")
+            self._authentication(id)
+        except CustomError as e:
+            self.bus.shutdown()
+            return e.message
+
+    def _write_by_identifier(self, id, identifier, value):
```
Comment (CatalinGP) [inline src/rest_api/actions/write_info_action.py:29]: make use of write_by_identifier from generate_frames.py

File: src/rest_api/actions/write_info_action.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            self._auth_mcu()
+            self._write_data(self.get_ecu_id())
+            response_json = self._to_json("success", 0)
+            return response_json
+        except CustomError as e:
+            self.bus.shutdown()
+            return e.message
+
+
+class WriteToDoors(WriteInfo):
```
Comment (CatalinGP) [inline src/rest_api/actions/write_info_action.py:85]: i thought we discussed to have methods inside WriteInfo class for write_to_bat, write_to_engine, write_to_doors...

**Review by CatalinGP — CHANGES_REQUESTED** [review]
please review my comments

**Review by CatalinGP — APPROVED** [review]

**Review by Mihnea9901 — APPROVED** [review]

**Pull request merged**